.travis.yml

```diff
@@ -16,23 +18,18 @@ addons:
       - build-essential
       - libudev-dev
 
-before_install:
-  - pkg-config --list-all
-  - pkg-config --libs libudev
-  - pkg-config --modversion libudev
-
 install:
-  - rustup component add rustfmt-preview
-  - rustup component add clippy-preview
+  - rustup component add rustfmt
+  - rustup component add clippy
 
 script:
 - |
-  if [ "$TRAVIS_RUST_VERSION" == "nightly" ] ; then
+  if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$TRAVIS_OS_NAME" == "linux" ] ; then
     export ASAN_OPTIONS="detect_odr_violation=1:leak_check_at_exit=0:detect_leaks=0"
     export RUSTFLAGS="-Z sanitizer=address"
   fi
 - |
-  if [ "$TRAVIS_RUST_VERSION" == "stable" ] ; then
+  if [ "$TRAVIS_RUST_VERSION" == "stable" ] && [ "$TRAVIS_OS_NAME" == "linux" ] ; then
```

Comment:
Do we not want to run rustfmt/clippy on windows?

Reply:
I was having timeouts trying it, there's probably more that needs doing. I'll open an issue for later.

Reply:
https://github.com/mozilla/authenticator-rs/issues/120